Dockerfile

```diff
@@ -9,6 +9,8 @@ ENV PYTHONHASHSEED 2157
 
 COPY . .
 
-RUN /bin/bash -c "source activate runenv && scripts/install_requirements.sh"
+RUN /bin/bash -c 'source activate runenv && scripts/install_requirements.sh &&\
+ pip install --no-cache-dir -q http://download.pytorch.org/whl/cu80/torch-0.1.11.post5-cp35-cp35m-linux_x86_64.whl &&\
+ conda install pytorch torchvision -c soumith -y -q'
```

Comment:
I think we can actually remove this conda line entirely if we aren't supporting end to end installation on OSX? I think Michael added this to get around running the docker image locally here: https://github.com/allenai/allennlp/pull/4/files#diff-731bfdf25ddc6c00e8273af674f3d041R7